sinks/dashboard/items/can_sender.py

```
@@ -112,7 +112,7 @@ def display_can_fields(self, fields: List[pf.Field]):
                 self.widget_widths[self.widget_index] = max_text_width + self.WIDGET_TEXT_PADDING
                 dropdown.setFixedWidth(self.widget_widths[self.widget_index])
                 self.widgets[self.widget_index] = dropdown
-            elif isinstance(field, pf.Numeric) or isinstance(field, pf.ASCII):
+            elif isinstance(field, pf.Numeric) or isinstance(field, pf.ASCII) or isinstance(field, pf.Bitfield):
```

Ruff (0.11.9)

Multiple `isinstance` calls for `field`, merge into a single call

Merge `isinstance` calls for `field`

(SIM101)